features/step_definitions/tag_steps.rb

```diff
@@ -374,10 +374,21 @@
   click_button("Save changes")
 end
 
+When "I assign the tag {string} to the fandom {string}" do |tag, fandom|
+  tag = Tag.find_by(name: tag)
+  visit edit_tag_path(tag)
+  fill_in("Fandoms", with: fandom)
+  click_button("Save changes")
+end
+
 When /^I view the (canonical|synonymous|unfilterable|unwrangled|unwrangleable) (character|relationship|freeform) bin for "(.*?)"$/ do |status, type, tag|
   visit wrangle_tag_path(Tag.find_by(name: tag), show: type.pluralize, status: status)
 end
 
+When /^I view the (character|relationship|freeform|fandom) mass bin$/ do |type| # rubocop:disable Cucumber/RegexStepName
+  visit tag_wranglings_path(show: type.pluralize)
+end
```

Comment:
Could we define these in support/paths.rb instead? The main benefit of that is it gives us "Given I am on the x mass bin page" and "When I go to the x mass bin page" and "Then I should be on the x mass bin page," which can all be useful.